pkg/cmd/server/server.go

```diff
@@ -242,112 +239,108 @@ func (s *server) watchConfig(config *api.Config) {
 
 func (s *server) initBackupService(config *api.Config) error {
 	glog.Infof("Configuring cloud provider for backup service")
-	cloud, err := initCloud(config.BackupStorageProvider.CloudProviderConfig, "backupStorageProvider")
+	objectStorage, err := getObjectStorageProvider(config.BackupStorageProvider.CloudProviderConfig, "backupStorageProvider")
 	if err != nil {
 		return err
 	}
-	s.backupService = cloudprovider.NewBackupService(cloud.ObjectStorage())
+
+	s.backupService = cloudprovider.NewBackupService(objectStorage)
 	return nil
 }
 
 func (s *server) initSnapshotService(config *api.Config) error {
+	if config.PersistentVolumeProvider == nil {
+		glog.Infof("PersistentVolumeProvider config not provided, skipping SnapshotService creation")
```

Comment:
Instead of saying "skipping SnapshotService creation", I would say "volume snapshots and restores are disabled"?